docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -343,50 +350,95 @@ The snippet below shows a part of a document with the annotated results:
+                "importance" : 0.9872151818111125,
+                "classes" : [
+                  {
+                    "class_name" : false,
+                    "importance" : 0.9872151818111125
+                  },
+                  {
+                    "class_name" : true,
+                    "importance" : -0.9872151818111125
+                  }
+                ]
+              },
+              ...
             ],
             "is_training" : false
           }
 ----
-<1> An array of values specifying the probability of the prediction and the 
-`class_score` for each class. 
+<1> An array of values specifying the probability of the prediction and the
+score for each class. 
+
+The class with the highest score is the prediction. In this example, `false` has
+a `class_score` of 0.39 while `true` has only 0.22, so the prediction will be
+`false`. For more details about these values, see
+<<dfa-classification-interpret>>.
+====
 
-The `top_classes` object contains the predicted classes with the highest 
-scores. The `class_probability` is a value between 0 and 1. The higher the 
-number, the more confident the model is that the data point belongs to the named 
-class. In the example above, `false` has a `class_probability` of 0.91 while 
-`true` has only 0.08, so the prediction will be `false`. The `class_score` is a 
-function of the probability.
+If you chose to calculate feature importance, the destination index also
+contains `ml.feature_importance` objects. Every field that is included in the 
+{classanalysis} (known as a _feature_ of the data point) is assigned a feature
+importance value. However, only the most significant values (in this case, the
+top 10) are stored in the index. These values indicate which features had the
+biggest impact (positive or negative) on each prediction. In {kib}, you can see
+this information displayed in the form of a decision plot:
 
+[role="screenshot"]
+image::images/flights-classification-importance.png["A decision plot for feature importance values in {kib}"]
+
+The sum of the feature importance values for a class (in this example, `false`)
+in this data point approximates the logarithm of its odds
+(or {wikipedia}/Logit[log-odds]).
 ////
-It is chosen so that the decision to assign the 
-data point to the class with the highest score maximizes the minimum recall of 
-any class.
+//Does this mean the sum of the feature importance values for false in this
+example should equal the logit(p), where p is the class_probability for false?
+//Does this imply that the feature importance value itself is the result of a logit function? Or that we use the function to merely represent the distribution of feature importance values?
 ////
-====
+While the probability of a class ranges between 0 and 1, its log-odds range 
+between negative and positive infinity. In {kib}, the decision path for each
+class starts near zero, which represents a class probability of 0.5.
```

Comment:
This is unfortunately a bit more complicated: 0 would represent the class probability of a constant baseline. It relates to the average class probability for the selected class (in Kibana UI) over entire training set.

If you select `Canceled` as the target variable in flight data, this nuance becomes obvious. Since there are many more data points with `Canceled = False`, let's assume that the average class probability over the entire training set would be something like `0.92`. This means that if the class probability of a data point is larger than `0.92` (for example `0.98`) than the decision path will go to the right (sum of feature importances is positive). On the other hand, if the class probability is smaller than `0.92` (for example 0.84), than the decision path will go to the left (sum of feature importances is negative).